chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/endpoints/ChaosMonkeyRequestScopeRestEndpointIntTest.java

```diff
@@ -99,11 +114,16 @@ void postChaosMonkeySettingsValueObjectWatcherPropertiesNULL() {
     // Watcher Tests
     @Test
     void getWatcherConfiguration() {
+        // openapi validation
+        testRestTemplate.getRestTemplate().setInterceptors(Collections.singletonList(validationInterceptor));
+
         ResponseEntity<WatcherProperties> result =
                 testRestTemplate.getForEntity(baseUrl + "/watchers", WatcherProperties.class);
 
         assertEquals(HttpStatus.OK, result.getStatusCode());
         assertEquals(chaosMonkeySettings.getWatcherProperties().toString(), result.getBody().toString());
+
+        testRestTemplate.getRestTemplate().setInterceptors(Collections.emptyList());
```

Comment:
I think this should be better done in tearDown. In case of an exception this wont be called/rolled back. The same could apply for setting the interceptors (in case its wanted for each test)

Reply:
Good point, removing interceptors added in df10163b99e532451b418a9e20ef30e46d588a7d.

Yes, adding the interceptors will be re-evaluated when I work over the whole file and move more things to OpenAPI (next PR, after you approve this one :-)).